config/settings.defaults.coffee

```diff
@@ -107,8 +117,8 @@ module.exports = settings =
 		# references:
 		# 	url: "http://localhost:3040"
 		notifications:
-			url: "http://localhost:3042" 
-			
```

Comment:
I think your editor trims whitespace on exit, which makes commits more noisey. Consider turning it off? (Don't worry for this commit) Or we should do a single pass to clean up whitespace and then add a linting rule or something.